### PR TITLE
chore: Update NEWS and debian/changelog

### PR DESCRIPTION
**Pull request by uristdwarf**
* Add tool for updating both, so you need to only write it once.
* Fix date in previous NEWS entry

### CODE REVIEW

File: utils/update_news.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+echo "Enter version (e.g., 4.0.2):"
+read version
+echo "Enter urgency (low, medium, high):"
+read urgency
+
+# Current date for changelog
+current_date=$(get_current_date)
+
+# Prepare temporary file for editing
+TMP_FILE="/tmp/changelog_$version"
```
Comment (Antuan-AE) [inline utils/update_news.sh:22]: Consider adding brackets around variables like: 
"${version}"

**Review by Antuan-AE — APPROVED** [review]
Please check comments regarding variables without brackets at update_news.sh. Aside from that it LGTM.

File: NEWS [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
+   Could cause a deadlock when reconnecting after a session was lost.
+ - (mount) Fix wrong GID being shown in .oplog
+ - (admin) Add `--help option` in saunafs-admin
+ - (admin) Add `--defaults` option for `dump-config` in saunafs-admin
+ - (cgi) Update saunafs-cgi.
+ - (all) Add support for Ubuntu 24.04
+ - (all) Fix compilation issues on GCC 13
+ - (all) Various bug fixes, especially regarding tests
+ - (windows) Support compiling common and client library
+ - (windows) Improve behavior for Windows client
```
Comment (lgsilva3087) [inline NEWS:17]: Consider adding:
- (mount) Improve DirEntryCache behaviour by caching getattr and lookup operations

**Review by lgsilva3087 — APPROVED** [review]
LGTM. Please see my suggestion.

File: utils/update_news.sh [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    # Check if line starts with '(' indicating a new item
+    if [[ $line == \(* ]]; then
+        news_entry+=" - $line\n"
+        changelog_entry+="  * $line\n"
+        previous_line_new_item=true
+    else
+        news_entry+="   $line\n"
+        changelog_entry+="   $line\n"
+    fi
+done < "$TMP_FILE"
```
[the remark this answers is not in the document]
Comment (uristdwarf) [reply utils/update_news.sh]: I couldn't get it to work :(

But we can always improve it later :)

I just quickly wrote this script so I or anyone else doesn't have to do it twice in two files.

**Pull request merged**